docs/configuration/extract-codacy-logs.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+#### Helper functions ####
+
+usage()
+{
+  echo "Usage: $0 -n NAMESPACE "
+  exit 2
+}
+
+set_variable()
+{
+  local varname=$1
+  shift
+  if [ -z "${!varname}" ]; then
+    eval "$varname=\"$@\""
```

Comment:
Why do you need eval?

Reply:
Anyway, in the beginning I had the expectation of using this more, but since it's only used once, I'll simplify.